server/pages/index.js

```diff
@@ -245,13 +246,15 @@ function getDefaultContext( request ) {
 		store: createReduxStore( initialServerState ),
 		bodyClasses,
 		sectionCss,
+		userAgent: useragent.parse( request.headers[ 'user-agent' ] ),
```

Comment:
`request.headers[ 'user-agent' ]` - Is this always set / on all browsers?

Reply:
Good question. It made me look this up.

Express [extends](https://github.com/expressjs/express/blob/b97faff6e2aa4d34d79485fe4331cb0eec13ad57/lib/request.js#L31) node's [HTTP module](https://nodejs.org/api/http.html#http_event_request). So a `headers` property will exist at least. 

But looking more closely, I realized I'd used `useragent.parse( request.headers[ 'user-agent' ] )` when I didn't need to at all.

We're adding [express-useragent](https://github.com/biggora/express-useragent) to the middleware in [boot.js](https://github.com/Automattic/wp-calypso/blob/master/server/boot/index.js#L30), which makes a parsed `req.useragent` object available on the request object.

Updated 👍